server/src/apub/community.rs

```diff
@@ -1,4 +1,5 @@
 use super::*;
+use activitystreams::actor::kind::GroupType;
```

Comment:
You're already messing up the imports lol. Tbh I dont really see the advantage of putting all the imports into mod.rs. Better install cargo nightly and use the fmt options I mentioned.

Reply:
Ah damnit good call. I moved this to the mod.rs for now, but yes I'll do another PR at some point for the import cleanups.